src/test/java/core/basesyntax/service/operationwithfruits/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service.operationwithfruits;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.service.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private StorageDao storageDao;
+    private FruitTransaction fruitTransaction;
+    private BalanceOperationHandler balanceOperationHandler;
+
+    @Before
+    public void setUp() {
+        storageDao = new StorageDaoImpl();
+        balanceOperationHandler = new BalanceOperationHandler(storageDao);
+        fruitTransaction = new FruitTransaction();
+        fruitTransaction.setOperation(fruitTransaction.convertOperation("b"));
+        fruitTransaction.setFruit("banana");
+    }
+
+    @Test
+    public void balanceSetPositive_Ok() {
+        fruitTransaction.setQuantity(10);
+        balanceOperationHandler.getOperation(fruitTransaction);
+        assertEquals(1,storageDao.getAllFruitsFromStorage().size());
+        assertTrue(storageDao.getAllFruitsFromStorage()
+                .containsKey(fruitTransaction.getFruit()));
+        assertTrue(storageDao.getAllFruitsFromStorage()
+                .containsValue(fruitTransaction.getQuantity()));
```

Comment:
```suggestion
        balanceOperationHandler.getOperation(fruitTransaction);
        Integer actual = Storage.storage.get(fruitTransaction.getFruit());
        Integer expected = fruitTranscation.getQuantity();
        assertEquals(expected, actual); 
```
Follow this principle in all tests concerning operations